src/ResourceManager/Common/Commands.ScenarioTests.ResourceManager.Common/AzureRM.Storage.ps1

```diff
@@ -27,14 +27,19 @@ function New-AzureRmStorageAccount
     [string] [Parameter(Position=0, ValueFromPipelineByPropertyName=$true)] $ResourceGroupName,
     [string] [Parameter(Position=1, ValueFromPipelineByPropertyName=$true)][alias("StorageAccountName")] $Name,
 	[string] [Parameter(Position=2, ValueFromPipelineByPropertyName=$true)] $Location,
-	[string] [Parameter(Position=3, ValueFromPipelineByPropertyName=$true)] $Type)
+    [Microsoft.Azure.Management.Storage.Models.AccountType] [Parameter(Position=3, ValueFromPipelineByPropertyName=$true)] $Type)
```

Comment:
I think this will break the current tests.
New-AzureRmStorageAccount takes a string with '_', for Type parameter, but this takes only strings without '_'.